swap_meet/vendor.py

```diff
@@ -0,0 +1,131 @@
+from swap_meet.item import Item
+
+class Vendor():
+
+    def __init__(self, inventory = None):
+
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+        
+
+    def add(self, item):  
+        self.inventory.append(item)
+        return item
+        
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            for i, element in enumerate(self.inventory):
+                if element == item:
+                    return self.inventory.pop(i)          
```

Comment:
You can just use the built in `list.remove`:

```suggestion
            self.inventory.remove(item)
```